backend/src/main/java/com/capstone/moneytree/service/impl/DefaultUserService.java

```diff
@@ -37,14 +37,16 @@
    private final AmazonS3Service amazonS3Service;
 
    private final String bucketName;
+   private final String defaultProfileURL;
 
    @Autowired
-   public DefaultUserService(UserDao userDao, ValidatorFactory validatorFactory, AmazonS3Service amazonS3Service, @Value("${aws.profile.pictures.bucket}") String bucketName) {
+   public DefaultUserService(UserDao userDao, ValidatorFactory validatorFactory, AmazonS3Service amazonS3Service, @Value("${aws.profile.pictures.bucket}") String bucketName, @Value("${aws.default.profile.picture}") String defaultProfileURL) {
       this.userDao = userDao;
```

Comment:
It does not make sense to pass a default url as constructor. You can inject it within the field attribute.

Reply:
right agreed, was coded at 2am 🤦‍♂️